### ISSUE TITLE
weird loss behavior

### ISSUE THREAD

**Issue by shiretzet** [fchollet/keras#2898]
I have 3 outputs, each have the same loss function 'categorical_crossentropy'.

The output of each loss is 0.4014 at every epoch, but the total (i think) loss is constantly changing (as needed).

Here is an example:

``` python
Epoch 1/5
500/500 [==============================] - 382s - loss: 5.1790 - Red_out_loss: 0.4014 - Green_out_loss: 0.4014 - Blue_out_loss: 0.4014 - Red_out_acc: 0.0041 - Green_out_acc: 0.0041 - Blue_out_acc: 0.0039   
Epoch 2/5
500/500 [==============================] - 388s - loss: 4.3626 - Red_out_loss: 0.4014 - Green_out_loss: 0.4014 - Blue_out_loss: 0.4014 - Red_out_acc: 0.0039 - Green_out_acc: 0.0044 - Blue_out_acc: 0.0035   
Epoch 3/5
500/500 [==============================] - 391s - loss: 4.3566 - Red_out_loss: 0.4014 - Green_out_loss: 0.4014 - Blue_out_loss: 0.4014 - Red_out_acc: 0.0039 - Green_out_acc: 0.0044 - Blue_out_acc: 0.0035   
Epoch 4/5
500/500 [==============================] - 393s - loss: 174.9229 - Red_out_loss: 0.4014 - Green_out_loss: 0.4014 - Blue_out_loss: 0.4014 - Red_out_acc: 0.0040 - Green_out_acc: 0.0041 - Blue_out_acc: 0.0039  
```

Y is that?

**Comment by joelthchao** [comment]
Please put your model and code in your question. Nobody can help you without that.
